From 53dc07994d9159304d3e5df6d833f1b0b3924676 Mon Sep 17 00:00:00 2001
From: Xu Qiang <xuqiang36@huawei.com>
Date: Mon, 13 Feb 2023 15:44:25 +0800
Subject: [PATCH 011/500] [Huawei] mm/sharepool: Delete unused tgid and spg_id
 in spg_proc_stat.

Offering: HULK
hulk inclusion
category: other
bugzilla: https://gitee.com/openeuler/kernel/issues/I6ET9W

----------------------------------------------

Signed-off-by: Xu Qiang <xuqiang36@huawei.com>
---
 mm/share_pool.c | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/mm/share_pool.c b/mm/share_pool.c
index 6627b306c9ff..8317a4c21d33 100644
--- a/mm/share_pool.c
+++ b/mm/share_pool.c
@@ -123,8 +123,6 @@ struct sp_meminfo {
 #ifndef __GENKSYMS__
 /* per process/sp-group memory usage statistics */
 struct spg_proc_stat {
-	int tgid;
-	int spg_id;  /* 0 for non-group data, such as k2u_task */
     /*
      * alloc amount minus free amount, may be negative when freed by
 	 * another task in the same sp group.
@@ -650,10 +648,8 @@ static void update_mem_usage_k2u(unsigned long size, bool inc,
 	}
 }
 
-static void sp_init_spg_proc_stat(struct spg_proc_stat *stat, int spg_id)
+static void sp_init_spg_proc_stat(struct spg_proc_stat *stat)
 {
-	stat->tgid = current->tgid;
-	stat->spg_id = spg_id;
 	atomic64_set(&stat->alloc_nsize, 0);
 	atomic64_set(&stat->alloc_hsize, 0);
 	atomic64_set(&stat->k2u_size, 0);
@@ -1276,7 +1272,7 @@ static struct sp_group_node *create_spg_node(struct mm_struct *mm,
 	spg_node->spg = spg;
 	spg_node->master = master;
 	spg_node->prot = prot;
-	sp_init_spg_proc_stat(&spg_node->instat, spg->id);
+	sp_init_spg_proc_stat(&spg_node->instat);
 
 	list_add_tail(&spg_node->group_node, &master->node_list);
 	master->count++;
-- 
2.17.1

